source/docs/software/old-commandbased/basics/creating-the-project.rst

```diff
@@ -4,3 +4,4 @@ Creating a Robot Project
 .. important:: This documentation describes the use of the legacy command-based library, which has been deprecated. While this documentation has been preserved to help teams that have yet to do so, teams are strongly encouraged to migrate to the :ref:`new command-based library <docs/software/commandbased/index:Command-Based Programming>`.
 
 Creating a project is detailed in :ref:`docs/software/wpilib-overview/creating-robot-program:Creating a Robot Program`. Select "Template" then "Command Robot" to create a basic Command-Based Robot program. Alternately you can use RobotBuilder to create the framework of your Command-Based Robot project as detailed in RobotBuilder.
+.. todo:: Link to RobotBuilder when it is added.
```

Comment:
This will cause the build-bot to fail. It may be worth to disable
```
# TODO Directives omit a warning
todo_emit_warnings = True
``` 
in the conf.py